examples/framesjs-starter/app/debug/components/frame-render.tsx

```diff
@@ -51,16 +52,27 @@ export function FrameRender({
                 );
                 return;
               }
-              return submitOption({
-                buttonIndex: index + 1,
-                inputText:
-                  frame.inputText !== undefined ? inputText : undefined,
-              });
+              if (action === "link") {
+                if (
+                  window.confirm("You are about to be redirected to " + target!)
+                ) {
+                  window.location.href = target!;
+                }
+              } else if (action === "mint") {
+                alert(`Requested to mint NFT: ${target}`);
```

Comment:
is there a way to post back post mint to the frame?

Reply:
spec says client should either compose a transaction or fall back to the frame url as an external link

Reply:
so no continued frame flow? hm will

Reply:
?